compiler-rt/lib/asan/asan_allocator.cpp

```diff
@@ -249,7 +249,14 @@ void AsanMapUnmapCallback::OnMap(uptr p, uptr size) const {
   thread_stats.mmaped += size;
 }
 void AsanMapUnmapCallback::OnUnmap(uptr p, uptr size) const {
+#if SANITIZER_CHEERPWASM
+  // CHEERP: as wasm is a linear memory model, trying to access unmapped
```

Comment:
"has a linear memory model" -> "has no memory protection within the program linear memory"?